r_basics_visualize_data/r_basics_visualize_data.md

```diff
@@ -0,0 +1,1064 @@
+<td><pre style = "color: rgba(var(--color-text), 0.5); margin:1em; font-size:0.8em;">
+ggplot(<span style = "color: rgb(var(--color-text)); font-weight: bold;">data = covid_testing</span>) +
+  geom_histogram(mapping = aes(x = pan_day))
+</pre>
+</td></tr>
+<tr style="padding: 3em; border: 1px solid rgb(var(--color-text));"><td>3) We build our plot across several different lines, so we include a <code>+</code> to say "wait, we're not done yet!"
+</td>
+<td><pre style = "color: rgba(var(--color-text), 0.5); margin:1em; font-size:0.8em;">
+ggplot(data = covid_testing) <span style = "color: rgb(var(--color-text)); font-weight: bold;">+ </span>
+  geom_histogram(mapping = aes(x = pan_day))
+</pre>
+</td></tr>
+<tr style="padding: 3em; border: 1px solid rgb(var(--color-text));"><td>4) In the second line, we describe what kind of geometric representation we want -- a histogram.
+</td>
+<td><pre style = "color: rgba(var(--color-text), 0.5); margin:1em; font-size:0.8em;">
+ggplot(data = covid_testing) +
+  <span style = "color: rgb(var(--color-text)); font-weight: bold;">geom_histogram(</span>mapping = aes(x = pan_day)<span style = "color: rgb(var(--color-text)); font-weight: bold;">)</span>
+</pre>
+</td></tr>
+<tr style="padding: 3em; border: 1px solid rgb(var(--color-text));"><td>5) We also add some mappings, explaining which data from the data frame should be displayed in the histogram.  We use "aes" (short for "aesthetic" or "aesthetic mapping") to tell ggplot how to draw the visualization.
+</td>
+<td><pre style = "color: rgba(var(--color-text), 0.5); margin:1em; font-size:0.8em;">
+ggplot(data = covid_testing) +
+  geom_histogram(<span style = "color: rgb(var(--color-text)); font-weight: bold;">mapping = aes(</span>x = pan_day<span style = "color: rgb(var(--color-text)); font-weight: bold;">)</span>)
+</pre>
+</td></tr>
+<tr style="padding: 3em; border: 1px solid rgb(var(--color-text));"><td>6) We only have to specify the <b>x axis</b>, because a histogram assumes that you're counting rows of data and will map that to the y axis.
+</td>
+<td><pre style = "color: rgba(var(--color-text), 0.5); margin:1em; font-size:0.8em;">
+ggplot(data = covid_testing) +
+  geom_histogram(mapping = aes(<span style = "color: rgb(var(--color-text)); font-weight: bold;">x = pan_day</span>))
+</pre>
+</td></tr>
+</table>
+</lia-keep>
+
+### A Template
+
+So that was a lot of information. Let's break this down to build up to a more general template. Here's a preview of that template, and we'll spend the few sections or so exploring it.
+
+![ggplot code annotated with the steps enumerated in the list below](media/ggplot_template.png)<!-- style = "max-width:700px;" -->
+
+You start with the code written in **bold** – **bold** in this template is the constant part - and fill in the details that match what you want. To fill in the details, you need to do 3 things:
+
+1) Pick a **tidy data frame** (this contains the data you want to plot)
```

Comment:
```suggestion
1) Pick a **tidy data frame** (this contains the data you want to plot, we'll go over what makes a data frame tidy in the next section)
```

Reply:
yes but also maybe move tidy data out?  I'll correct for now and then look at the overall gestalt